libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -430,6 +474,47 @@ bool AP_Arming::battery_checks(bool report)
                 }
                 return false;
             }
+
+            float temperature = 0.0f;
+            // temperature minimum check
+            if (_min_temperature[i] > 0.0f) {
+                if (!_battery.get_temperature(temperature, i)) {
+                    if (report) {
+                        gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: Battery %d not get the temp",
```

Comment:
`PreArm: No temperature for battery %u`

Reply:
I will change the message to "PreArm: No temperature for battery %u".